ui/css/perf.css

```diff
@@ -493,6 +493,10 @@ li.pagination-active.active > button {
 }
 
 /* table in graphs view */
+.text-muted {
+  color: #53595f;
```

Comment:
This color does pass the contrast ratio, but it is not reflected on the UI. `text-muted` is a Bootstrap class, so it would only work if you add `!important`, but we won't do that. I will add a comment with a suggestion on what to do.